src/app/terms/page.js

```diff
@@ -0,0 +1,86 @@
+      <Box sx={{
+        backgroundColor: theme.palette.background.paper,
+        fontFamily: theme.typography.fontFamily,
+        p: theme.spacing(4),
+        mt: theme.spacing(5),
+        mb: theme.spacing(5),
+      }}>
+        <Box sx={{ backgroundColor: theme.palette.grey[200],
+          textAlign: 'center',
+          mb: theme.spacing(4),
+          p: theme.spacing(14),
+          borderRadius: theme.shape.borderRadius
+         }}>
+          <Typography variant="h3">
+            Terms and conditions
+          </Typography>
+          <Typography variant="body1">
+            Last updated: 21 September 2023
+          </Typography>
+        </Box>
+        <Typography variant="body1" paragraph>
+          These Terms and Conditions (the "Terms") describe the terms and conditions on which OOSCCA Ltd., a United Kingdom company (“OOSCCA,” “we,” “us,” or “our”) makes its website (“Site”) and application (collectively the “Services”) available to you. By using or accessing our Services, you confirm your agreement to be bound by these Terms and our Privacy Policy, which are incorporated herein by reference. If you do not agree to these Terms and the Privacy Policy, please do not use or access the Services. The Terms expressly supersede any prior terms of use between you and us or any of our affiliates or predecessors.
+        </Typography>
+        <Typography variant="h6">
+          Acceptance of the Terms
+        </Typography>
+        <Typography variant="body1" paragraph>
+        OOSCCA's products, features and offerings are available made available through www.OOSCCA.com and app.OOSCCA.com. All of these are collectively referred to as "OOSCCA Properties" or our "Services". The material, including without limitation information, data, text, editorial content, design elements, look and feel, formatting, graphics, images, photographs, videos, music, sounds and other content contained in or delivered via the Services or otherwise made available by OOSCCA in connection with the Services is the "Site Content" (or "Content"). Any material (including the foregoing categories) that you contribute, provide, post or make available using the Services is "Your Content."
+        </Typography>
+        <Typography variant="body1" paragraph>
+          When these Terms use the term "Organiser," we mean event creators using the Services to create events displayed on the Services for consumers using our Services (a) to consume information about or attend Events ("Consumers"), or (b) for any other reason. Organisers, Consumers and third parties using our Services are all referred to in these Terms collectively as "Users," "you" or "your."
+          When these Terms use the term "OOSCCA," "we," "us," or "our," that refers to OOSCCA, Inc. and its affiliates, and subsidiaries, and each of its and their respective officers, directors, agents, partners and employees.
+        </Typography>
+        <Typography variant="body1" paragraph>
```

Comment:
it is realy hard to read. I would divide it into several sections - separate header, etc.